image/templates/helm/stackrox-secured-cluster/README.md.htpl

```diff
@@ -127,7 +127,7 @@ For example,
   ```
 - **Configure collection method**:
   ```sh
-  --set collector.collectionMethod=EBPF
+  --set collector.collectionMethod=CORE_BPF
```

Comment:
This is really just a nit, but maybe it would make sense to stick with `EBPF` here since `CORE_BPF` is now the default and a user looking at the docs and the example parameters might be interested in changing this to `EBPF`?

Reply:
Likely no, it's not what we want. The reason is that `EBPF` as a collection method is going to be deprecated, and if there still would be any mention of it in the readme, it would only cause confusion and unnecessary questions. In fact, in the nearest future we plan to remove the `collectionMethod` configuration altogether, leaving it to the Collector to decide which method to use. Does it make sense to you?